packages/react/src/experimental/Await.spec.tsx

```diff
@@ -96,3 +96,30 @@ describe('useAwait', () => {
     await waitFor(() => expect(screen.queryByText(TEXT)).toBeInTheDocument())
   })
 })
+
+describe('Await component', () => {
+  it('should render child component with data from useAwait hook', async () => {
+    render(
+      <Suspense fallback="Loading...">
+        <Await options={{ key, fn: () => Promise.resolve(TEXT) }}>{({ data }) => <>{data}</>}</Await>
+      </Suspense>
+    )
+
+    expect(await screen.findByText(TEXT)).toBeInTheDocument()
+  })
+})
+
+describe('class AwaitClient clearError', () => {
+  beforeEach(() => awaitClient.reset())
+
+  it('clears promise & error for all cache without key', async () => {
+    try {
+      await awaitClient.suspend({ key, fn: () => Promise.reject(new Error(ERROR_MESSAGE)) })
+    } catch {
+      // Expected an error
+    }
+
+    awaitClient.clearError()
+    expect(awaitClient.getData([key])).toBeUndefined()
```

Comment:
This may be proper code

```suggestion
    expect(awaitClient.getData(key)).toBeUndefined()
```